games-emulation/hatari/files/hatari-2.1.0_caps5_files.patch

```diff
@@ -0,0 +1,147 @@
+diff -Nuar hatari-2.1.0-orig/cmake/FindCapsImage.cmake hatari-2.1.0/cmake/FindCapsImage.cmake
```

Comment:
What's the source of this patch? Is it a backport from upstream VCS? Please include some link to the source on top of the file so others can verify.

Reply:
I created this patch myself to current tip and reformatted to 2.1.0. Committing to upstream in progress.